# Service to restore robot to default configuration

---
int64 ret
string msg
